### PR TITLE
Germline_CNV_allow_moderate_amplifications

### PR DESCRIPTION
**Pull request by TammushB**

### CODE REVIEW

File: ugvc/cnv/cnv_calling_using_cnmops.R [hunk 1]
```diff
@@ -33,6 +33,8 @@ parser$add_argument("--save_hdf", action='store_true',
                     help="whether to save additional data-frames in hdf5 format")
 parser$add_argument("--save_csv", action='store_true',
                     help="whether to save additional data-frames in csv format")
+parser$add_argument("--moderate_amplificiations", action='store_true',
+                    help="whether to call moderate amplifications (Fold-Change>0.2 & < 0.5 will be tagged as CN2.5)")
```
Comment (ilyasoifer) [inline ugvc/cnv/cnv_calling_using_cnmops.R]: Can you please state the fold change not in log scale?

File: ugvc/cnv/cnv_calling_using_cnmops.R [hunk 2]
```diff
@@ -41,10 +43,18 @@ cores <- args$parallel
...
-saveRDS(resCNMOPS_Int,file="cohort.cnmops.rds")
+if(args$moderate_amplificiations)
+{
+    resCNMOPS <- cn.mops(merged_cohort_reads_count,parallel=cores,minWidth = as.integer(min_width_val),norm=0,I = c(0.025,0.5,1,1.2,1.5,2,2.5,3,3.5,4),classes=c("CN0","CN1","CN2","CN2.5","CN3","CN4","CN5","CN6","CN7","CN8"),upperThreshold=0.2,lowerThreshold=-0.5,moderate_amplifications=TRUE)
+    saveRDS(resCNMOPS,file="cohort.cnmops.resCNMOPS.rds")
+    resCNMOPS_Int <-calcIntegerCopyNumbers(resCNMOPS)
+    saveRDS(resCNMOPS_Int,file="cohort.cnmops.resCNMOPS_Int.rds")
+}else {
+    resCNMOPS <- cn.mops(merged_cohort_reads_count,parallel=cores,minWidth = as.integer(min_width_val),norm=0,I = c(0.025,0.5,1,1.5,2,2.5,3,3.5,4),classes=c("CN0","CN1","CN2","CN3","CN4","CN5","CN6","CN7","CN8"))
+    saveRDS(resCNMOPS,file="cohort.cnmops.resCNMOPS.rds")
```
Comment (ilyasoifer) [inline ugvc/cnv/cnv_calling_using_cnmops.R]: Can you say what is the difference between the two RDS?

**Review by ilyasoifer — COMMENTED** [review]
Hi, looks good - just two comments

File: ugvc/cnv/cnv_calling_using_cnmops.R (hunk 2 above)
Comment (TammushB) [reply ugvc/cnv/cnv_calling_using_cnmops.R]: resCNMOPS and resCNMOPS_Int are almost the same. the _Int object has the fold-change translated to integer copy-number. I outputted both for debugging. keeping resCNMOPS_Int only.

**Review by ilyasoifer — APPROVED** [review]

**Pull request merged**